src/Time/TimeSteppers/AdamsLts.cpp

```diff
@@ -87,6 +88,15 @@ LtsCoefficients& add_assign_impl(LtsCoefficients& a, const LtsCoefficients& b,
       ++write;
     }
   }
+  {
+    auto write = a.begin();
+    for (const auto& entry : a) {
+      if (get<2>(entry) != 0.0) {
```

Comment:
Is there any reason to think that these numbers might be approximately zero to the level we don't care about?

Reply:
You're right, these aren't exact.  This should have a roundoff check of some sort.